components/Header.js

```diff
@@ -26,15 +25,8 @@ const StyledSun = styled(Sun)`
   fill: var(--main-dark);
 `;
 
-const StyledMenuButton = styled(Menu)`
-  width: 2.5rem;
-  fill: var(--main-dark);
-  border-style: none;
-`;
-const StyledCloseButton = styled(Close)`
-  width: 2.5rem;
-  fill: var(--contrast-text);
-  border-style: none;
+const StyledMenu = styled.div`
+  color: ${({ $isOpen }) => $isOpen};
```

Comment:
"$isOpen" might not be the best name for a color 🙂

Reply:
Yes, you're right :) Thank you!